### ISSUE TITLE
No Bot response, Empty Message Text

### ISSUE THREAD

**Issue by TenMediaBln** [botman/botman#669]
- BotMan Version: 2.1.6
- PHP Version: 7.0
- Messaging Service(s): Telegram
- Cache Driver: file

### Description:

Hi guys!

Thank you for providing such a great package!

Everything worked very well, until I installed the first driver (Telegram) and wanted to receive my bots request. I got no response, even though my system was seemingly working without any problem.

Digging deeper I found that I got a 400 response from Telegram saying “Bad Request: message text is empty”. 

Long story short, caused by Curl.php Line 26 http_build_query() . It encoded & to &amp; on my machine (PHP 7). After setting the separator manually everything is working fine. 

http_build_query($postParameters, null, '&')

I’m not sure, if this is an PHP 7 related issue, config issue or where it comes from. 

I hope it helps someone.

**Comment by mpociot** [comment]
Hi!

By default, this method uses the php.ini setting “arg_separator.output” which defaults to “&”. 
Can you check your local php.ini?
Maybe it might make sense to add this as a default in curl.php in case PHP is configured differently.

**Comment by TenMediaBln** [comment]
Yes, you were right. "arg_separator.output" was set to &amp;. Thank you!
